aws/build.rake

```diff
@@ -233,6 +233,17 @@ $websites_test = build_task('websites-test', [deploy_dir('rebuild')]) do
     end
   end
 
+  Dir.chdir(shared_dir) do
+    HipChat.log 'Running <b>shared</b> unit tests...'
```

Comment:
It's `shared` because these are rake tests for the /shared dir.  Right now there is only coverage for the channels API, but in the future we could add more.